### DISCUSSION TITLE
Which version(s) of libfmt is spdlog compatible with?

### DISCUSSION THREAD

**Discussion by lalten** [gabime/spdlog#2995] [Q&A] [answered]
Which version(s) of libfmt is spdlog compatible with?

It looks like the bundled libfmt was upgraded to v10 in https://github.com/gabime/spdlog/commit/62e55e7a7fb2fd3fd43fced10cccf166fcd8ff40 only to be reverted again in https://github.com/gabime/spdlog/commit/4338b9cd23c90c7bacde5bfefddade8d02dfd821.

The Bazel Central Registry currently defines a dependency on libfmt 10 in https://github.com/bazelbuild/bazel-central-registry/blob/90ea6ab1277d1f338db9204d56276409b812ac99/modules/spdlog/1.12.0/MODULE.bazel#L7. So I'm wondering if this setup is officially supported.

Cc @mvukov (https://github.com/mvukov/rules_ros2/pull/238#discussion_r1467311017) @vertexwahn (https://github.com/bazelbuild/bazel-central-registry/pull/1206)

**Comment by tt4g** [comment]
Downgraded to 9 after upgrading to 10 due to compatibility issues: #2767
However, this commit 0ca574ae168820da0268b3ec7607ca7b33024d05 should be compatible with the fmt 10.

**Reply by lalten** [reply, marked as answer]
Ah, I'm seeing that in the release notes for https://github.com/gabime/spdlog/releases/tag/v1.12.0 as well now.
So spdlog 1.11 only supports fmt 9.
Is my understanding correct that 1.12 and 1.13 both support (external) fmt 10?

**Reply by tt4g** [reply]
Right.
